packages/components/organisms/f-header/src/components/NavLink.vue

```diff
@@ -7,10 +7,15 @@
             [$style['c-navLink']]: !isPopoverLink,
             [$style['c-navLink--popoverLink']]: isPopoverLink,
             [$style['c-nav-list-link--alt']]: isAltColour,
+            [$style['c-navLink--condensedMid']]: isCondensed,
```

Comment:
Sorry for the post-merge comment – with the update to the prop, probably makes sense to update the classname to just be `c-navLink--condensed` as well?

Reply:
No worries, yea I spotted that after merging too, I've got another upcoming piece of work to do with f-header, so I'll make the change there, I'll also move the `hide-on-mid` class into fozzie as per @jamieomaguire's comment 😄 Alternatively I'll raise a patch PR for these small changes if I don't get around to that work soon. 😄

Reply:
Nice one 👍